### DISCUSSION TITLE
Copilot refuses to connect on VS code - Extension activation failed: "command 'codequote.openMatchPanel' already exists"

### DISCUSSION THREAD

**Discussion by abhishek-ib** [community/community#80831] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

[INFO] [copilotIgnore] [2023-12-18T21:28:32.021Z] inactive
[INFO] [codequote] [2023-12-18T21:28:32.021Z] CodeQuote enabled
[INFO] [copilotIgnore] [2023-12-18T21:28:32.080Z] inactive
[INFO] [codequote] [2023-12-18T21:28:32.080Z] CodeQuote enabled
[ERROR] [default] [2023-12-18T21:28:32.081Z] GitHub Copilot could not connect to server. Extension activation failed: "command 'codequote.openMatchPanel' already exists"

**Comment by ArianKhorasani** [comment]
I'm getting the same error! I did reinstall the extension several times and also signed out many times, but the error still exists! I have also tried to upgrade VS code but nothing changed!

**Comment by abhishek-ib** [comment, marked as answer]
Downgrading to the older version helps. 
Go to Extensions >  Copilot > Choose an older version > Restart VS Code and log in again.

I downgraded to version 1.84.51, and it works now.

Or upgrade your VS code.

**Reply by ArianKhorasani** [reply]
Thank you for your prompt reply @abhishek-ib! Just FYI, I tried a different way! I upgraded my VS code version to 1.85.1 and then I see it is working now!

**Reply by alperari** [reply]
Thanks, i had the same issue on my VSCode on WSL. 
It works on Windows but not on WSL.
Downgraded to `v1.122.462` and it works now 👍

**Comment by miriamglzr** [comment]
I have the same problem since a month from now. I paid without using it.

either
GitHub Copilot could not connect to server. Extension activation failed: "command 'codequote.openMatchPanel' already exists"

or

 GitHub Copilot could not connect to server. Extension activation failed: "Cannot read properties of undefined (reading 'onDidPerformUserAction')"
 
 I am considering to end my subscription

**Reply by miriamglzr** [reply]
I downgraded to v1.82.15 which was the first that seemed to work. But they should fix this.

**Reply by gyf3154** [reply]
me too

**Reply by alvin528** [reply]
thank you. It works for me.

**Reply by chenshuai088** [reply]
yes 1.82.15 is ok

**Reply by wsj97** [reply]
Thanks so much, it works for me too!!!

**Reply by gaowqspace** [reply]
1.82.15 is ok, thank you.

**Comment by SMGolamAlif** [comment]
Hey abhishek-ib,

Looks like Copilot is playing hard to get with that "command 'codequote.openMatchPanel' already exists" error. Here's a quick playbook to troubleshoot:

Check for Extension Conflicts: The error suggests there might be a clash with another extension. Check if you have any extensions that might be using the same command. Try disabling other extensions one by one to identify the culprit.

Reinstall Copilot: Uninstall the Copilot extension, restart VS Code, and reinstall it. Sometimes, a clean slate can do wonders.

Update Extensions: Ensure all your extensions, including Copilot, are up to date. An outdated extension might throw a wrench into the works.

GitHub Authentication: Confirm your GitHub credentials and ensure you're logged in. Copilot might be having trouble connecting to the server due to authentication issues.

Check GitHub Status: Occasionally, GitHub services might face hiccups. Check the GitHub status page to ensure everything's running smoothly.

If the issue persists, diving into VS Code's GitHub Copilot GitHub repository discussions might yield more specific insights. Others might have faced and resolved similar issues.

Hope this gets you back in the coding groove! 🚀💻

**Reply by HarshXAI** [reply]
yes , this works tysm

**Reply by cooperbrown9** [reply]
Updating my VS Code and then reinstalling copilot worked for me

**Comment by tskj** [comment]
Downgrading to `1.82.x` solved it for me, thanks for suggesting! Weird that they're not able to fix this.

**Comment by chukalicious** [comment]
Downgrading worked for me.

**Reply by mukhil0212** [reply]
how to downgrade it

**Reply by alperari** [reply]
@mukhil0212 from extensions page there is a dropdown next to uninstall
![image](https://github.com/community/community/assets/68128434/de5b7d19-72ed-4f6e-8189-8e0875f4227a)

**Reply by huynq-fouj** [reply]
hi

**Comment by huynq-fouj** [comment]
hi
